ansible/testbed-cli.sh

```diff
@@ -116,6 +117,21 @@ function renumber_topo
   echo Done
 }
 
+function add_config
```

Comment:
>add_config [](start = 9, length = 10)

This function has nothing to do with 'testbed'. Suggest remove it. Since it is quite simple, there is no need to wrap a script around the playbook. Add the ansible command line as a comment to the YML file.